src/Survey/ItechArt.Survey.Foundation/Validation/Abstractions/IValidator.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ItechArt.Survey.Foundation.Validation.Abstractions;
+
+public interface IValidator<TValidationObject, TValidationOptions, TValidationResult>
```

Comment:
`IValidator<T, TError> where TError : enum`
`ValidationResult<TError> Validate(T entity)`

Reply:
move to `Common` project (along with ValidationResult)